homeassistant/components/anova/sensor.py

```diff
@@ -0,0 +1,113 @@
+        icon="mdi:clock-outline",
+        name="Cook Time",
+    ),
+    SensorEntityDescription(key=AnovaPrecisionCookerSensor.STATE, name="State"),
+    SensorEntityDescription(key=AnovaPrecisionCookerSensor.MODE, name="Mode"),
+    SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.TARGET_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer",
+        name="Target Temperature",
+    ),
+    SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.COOK_TIME_REMAINING,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        icon="mdi:clock-outline",
+        name="Cook Time Remaining",
+    ),
+    SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.HEATER_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer",
+        name="Heater Temperature",
+    ),
+    SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.TRIAC_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer",
+        name="Triac Temperature",
+    ),
+    SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.WATER_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer",
+        name="Water Temperature",
+    ),
+]
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: config_entries.ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Anova device."""
+    anova_data: AnovaData = hass.data[DOMAIN][entry.entry_id]
+    sensors: list[AnovaSensor] = []
+    for coordinator in anova_data.coordinators:
+        await coordinator.async_config_entry_first_refresh()
+        sensors.extend(
+            [
+                AnovaSensor(coordinator, description)
+                for description in SENSOR_DESCRIPTIONS
+            ]
+        )
+    async_add_entities(sensors)
+
+
+class AnovaSensor(AnovaEntity, SensorEntity):
+    """A sensor using anova coordinator."""
+
+    def __init__(
+        self, coordinator: AnovaCoordinator, description: SensorEntityDescription
+    ) -> None:
+        """Set up an Anova Sensor Entity."""
+        super().__init__(coordinator)
+        self.entity_description = description
+        self._sensor_update_key = description.key
+        self._sensor_data = None
+        self._attr_unique_id = (
+            f"{coordinator._device_unique_id}_{description.key}".lower()
+        )
```

Comment:
```suggestion
        self._attr_unique_id = f"{coordinator._device_unique_id}_{description.key}"
```

It doesn't look like this needs to be `lower`